tests/src/test/scala/runtime/actionContainers/NodeJsActionContainerTests.scala

```diff
@@ -176,7 +177,10 @@ abstract class NodeJsActionContainerTests extends BasicActionRunnerTests with Ws
       initCode should be(200)
 
       val (runCode, runRes) = c.run(runPayload(JsObject()))
-      runCode should not be (200)
+      // actionloop proxy does not return a different error code when there is an error,
+      // because it communicates only through json
```

Comment:
i dont understand this comment - communicates with the HTTP caller? so the status code is always 200 from the proxy?

Reply:
yes it is and it is a long standing "issue", but it is not really an issue since the controller sets a different error code if the body is "error". Already discussed a few times.